plugins/module_utils/network/nxos/config/l2_interfaces/l2_interfaces.py

```diff
@@ -143,7 +169,7 @@ def set_state(self, want, have):
             commands.extend(self._state_deleted(want, have))
         else:
             for w in want:
-                if state == "merged":
+                if state in ["merged", "rendered"]:
```

Comment:
if state in ("overridden", "merged", "replaced") and not want:  rendered is missing.